pombola/core/static/js/hover-dropdown.js

```diff
@@ -0,0 +1,69 @@
+$(function(){
+
+    /*
```

Comment:
Yep, yui-compressor does remove single line comments, so I'll make that change and fixup that commit.